content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -502,7 +545,7 @@ and an empty `status.typeChecking` means that no errors were detected.
 For example, given the following policy definition:
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?